gossip/components/processor.c

```diff
@@ -43,7 +43,8 @@ static retcode_t process_transaction_bytes(processor_t const *const processor, t
                                            flex_trit_t const *const hash) {
   retcode_t ret = RC_OK;
   bool exists = false;
-  iota_transaction_t transaction = {.metadata = {.snapshot_index = 0, .solid = 0}, .loaded_columns_mask = {0}};
+  iota_transaction_t transaction;
+  memset(&transaction, 0, sizeof(transaction));
```

Comment:
Can you call `transaction_reset` on the transaction instead ? And do it after the variables declaration, maybe right after the `if` that checks the parameters

Reply:
Yes, modified. Thank you for pointing this out.